diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,37 @@
+```plantuml
+@startuml
+
+autonumber
+
+participant "Wallet" as w
+
+participant "User Agent" as u
+
+participant "Verifier" as r
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request_uri, request_uri_method=POST, [client_id_scheme])
+deactivate r
+u --> w: authorization request\n(client_id, request_uri, request_uri_method=POST, [client_id_scheme])
+activate w
+w --> r: POST **request_uri** (\n[OPTIONAL]wallet_metadata, \n[OPTIONAL]wallet_nonce)
+r -> r: create and sign (and optionally encrypt) request object 
```

Comment:
Is signing always required? If it is, then why is `wallet_nonce` optional? I think there might be cases where you don't want to sign, e.g., if encrypted responses are enabled and trust is established based on the domain.

Reply:
the sequence diagram is none normative, implementers can do what they want as long as it is compliant with RFC 9101. It requires a JWT, but would allow for alg `none`

Reply:
Lets not encourage "alg: none" in new specs... If you need to return unsecured content, return it as plain JSON.

If you need to send unsecured content, send it as plain JSON.

Reply:
discussion on the WG call today: agreement that an unsigned request (in plain JSON) is desirable. We will leave the PR as is and discuss the design of this extension in a new issue.